omero/sysadmins/in-place-import.rst

```diff
@@ -4,7 +4,7 @@ In-place import
 ===============
 
 In-place import allows files which are already present on the server machine
-to be imported into OMERO without the need to copy them. This requires
+to be imported into OMERO without the need to copy them into OMERO's ManagedRepository. This requires
```

Comment:
Confusing given `--transfer=cp`. Perhaps,

> ... without the need to upload them over an OMERO session.

Reply:
Suggestion?

Reply:
Sure, is just a suggestion.

Reply:
Sorry, I meant, what change would you suggest?

Reply:
Oh, @jburel explains, sorry - I didn't yet make time to experiment with them, e.g., no idea yet how useful it is to just give a *fragment* of the line or what. Planning to get around to trying.

Reply:
My suggestion's after my "perhaps"? We might be talking completely at cross-purposes, can chat if that'd help?

Reply:
`transfer=cp` is nearly a contradiction in terms, and not mentioned on this page. If you have a phrasing that you think is still generally understandable without making the `cp` case _more_ confusing, I'm all ears.